test-common/test-common-internal-api/src/main/java/org/apache/kafka/common/test/api/README.md

```diff
@@ -104,7 +104,7 @@ For each generated invocation:
 * Non-static `@AfterEach` methods are called
 * Static `@AfterAll` methods are called
 
-`@BeforeEach` methods give an opportunity to setup additional test dependencies before the cluster is started. 
```

Comment:
please update the above comments 
```
* Static `@BeforeAll` methods are called
* Test class is instantiated
* Non-static `@BeforeEach` methods are called
* Kafka Cluster is started
* Test method is invoked
* Kafka Cluster is stopped
* Non-static `@AfterEach` methods are called
* Static `@AfterAll` methods are called
```

Reply:
I rewrote a lot of this readme since it was kind of outdated.